src/util/worker.ts

```diff
@@ -231,9 +231,13 @@ export class PageWorker {
           break;
         }
 
-        if (retry >= 1000000000) {
+        if (retry >= 3) {
```

Comment:
I wonder if we even want to set this lower, like 2 by default? Can do that in separate issue/PR though if we want.

Reply:
Yeah, maybe it should be 2 really, since this generally shouldn't be timing out..